### ISSUE TITLE
Error  when run jsc files

### ISSUE THREAD

**Issue by salehmosleh** [OsamaAbbas/bytenode#58]
i compiled below code by bytenode CLI : `bytenode --compile test.js`
`console.log("test!!!");`
and when i run test.jsc output file by `bytenode test.jsc` in CLI i get below error:

```
C:\Users\saleh mosleh\AppData\Roaming\npm\node_modules\bytenode\index.js:179
    dummyCode = '"' + "\u200b".repeat(length - 2) + '"'; // "\u200b" Zero width space
                               ^

RangeError: Invalid string length
    at String.repeat (native)
    at Object.Module._extensions.(anonymous function) [as .jsc] (C:\Users\saleh mosleh\AppData\Roaming\npm\node_modules\bytenode\index.js:179:32)
    at Module.load (module.js:545:32)
    at tryModuleLoad (module.js:508:12)
    at Function.Module._load (module.js:500:3)
    at Function.Module.runMain (module.js:665:10)
    at startup (bootstrap_node.js:187:16)
    at bootstrap_node.js:607:3
```

**Comment by s100** [comment]
Works for me... please can you share

* exactly what platform and version of Node.js you are using
* exactly what version of `bytenode` you are using

**Comment by salehmosleh** [comment]
@s100 , i'm using windows 10 with below  node and bytenode version:
```
E:\nodejs\organchart-elc>node -v
v8.6.0
E:\nodejs\organchart-elc>npm -v
6.14.4
E:\nodejs\organchart-elc>bytenode -v
bytenode 1.1.6
```
Yes, exactly same.

**Comment by s100** [comment]
I have duplicated your issue. I think the problem you're having is very likely to be due to the mismatch between the levels of Node.js and npm you're using. [Node.js v8.6.0 shipped with npm v5.3.0](https://nodejs.org/en/download/releases/) - it actually took some effort for me to reproduce this environment since `nvm` won't do it. npm version 6.14.4, by comparison, shipped with Node.js 13.12.0, barely a month and a half ago. Your steps work for me with both of those combinations.

Additionally, Node.js v8.x.x ended long term support on 31 December 2019. So I suggest upgrading to a supported version of Node.js and trying again.

**Comment by salehmosleh** [comment]
@s100 , thank you, i updated nodeJS and it works fine!
